source/mobile/mobile-troubleshoot-notifications.rst

```diff
@@ -0,0 +1,23 @@
+Troubleshooting Push Notifications
+==================================
+
+If you did not receive a push notification when :doc:`testing push notifications <mobile-testing-notifications>`, use the following procedure to troubleshoot:
+
+1. Under **System Console** > **General** > **Logging** > **File Log Level** select **DEBUG** in order to watch for push notifications in the server log.
+
+2. Delete your mobile application, and install it again.
+
+3. Sign in with "Account A" and **confirm you want to receive push notifications** when prompted by the mobile app.
+
+4. On desktop, go to **Account Settings** > **Security** > **View and Logout of Active Sessions** and check that there is a session for the native mobile app matching your login time.
+
+5. Repeat the procedure for :doc:`testing push notifications <mobile-testing-notifications>`.
+
+6. If no push notification appears go to **System Console** > **Logs** and click **Reload**. Look at the bottom of the logs for a message similar to:
+
+``[2016/04/21 03:16:44 UTC] [DEBG] Sending push notification to 608xyz0... wi msg of '@accountb: Hello'``
```

Comment:
I realise this is a log message but is "... wi msg..." correct?  What does "wi" stand for?

Reply:
This should be a direct copy of the log message (other than the shortened userID). my assumption is that it means "with" in this context.